Fix:
error: implicit declaration of function 'puts' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'getpagesize' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'rand' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'umask' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'write' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'close' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'read' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'unlink' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2014-08-11 14:14:36.000000000 -0500
+++ configure	2022-08-20 23:47:00.000000000 -0500
@@ -913,7 +913,8 @@
   cat > conftest.$ac_ext <<EOF
 #line 915 "configure"
 #include "confdefs.h"
+#include <stdio.h>
 
 int main() {
 extern char *__progname; 
 	puts(__progname)
@@ -2046,6 +2047,9 @@
 #include <sys/types.h>
 #include <fcntl.h>
 #include <sys/mman.h>
+#include <sys/stat.h>
+#include <stdlib.h>
+#include <unistd.h>
 
 /* This mess was copied from the GNU getpagesize.h.  */
 #ifndef HAVE_GETPAGESIZE
--- configure.orig	2024-10-03 14:40:57
+++ configure	2024-10-03 14:43:29
@@ -761,7 +761,7 @@
 #line 762 "configure"
 #include "confdefs.h"
 
-main(){return(0);}
+int main(void) {return(0);}
 EOF
 if { (eval echo configure:767: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
   ac_cv_prog_cc_works=yes
@@ -881,7 +881,7 @@
   echo $ac_n "(cached) $ac_c" 1>&6
 else
   ac_cv_lbl_static_flag=unknown
-    echo 'main() {}' > conftest.c
+    echo 'int main(void) {return 0;}' > conftest.c
     if test "$GCC" != yes ; then
 	    trial_flag="-Bstatic"
 	    test=`$CC $trial_flag -o conftest conftest.c 2>&1`
